ui/src/shared/components/HistogramChartBars.tsx

```diff
@@ -205,6 +188,14 @@ class HistogramChartBars extends PureComponent<Props, State> {
     })
   }
 
+  private handleBarClick = (group: BarGroup) => (): void => {
+    const {onBarClick} = this.props
+
+    if (onBarClick) {
+      onBarClick(group)
```

Comment:
Instead of passing `group`, can we pass `group.data` (of type `HistogramData`)? Or even better, just the `HistogramDatum` of the sub-bar that was clicked?

The `BarGroup` interface contains implementation details that are tightly coupled to the `HistogramChartBars` component. By exposing it via the component `onBarClick` prop, we break the encapsulation of the component. I'd like to move the `BarGroup` interface back as a non-exported member of this module as well.